app/controllers/organisations_controller.rb

```diff
@@ -2,7 +2,9 @@ class OrganisationsController < ApplicationController
   before_action :set_organisation, only: %i[edit update]
   before_action :validate_user_is_part_of_organisation, only: %i[edit update]
 
-  def index; end
+  def index
+    render :show
```

Comment:
Why do we need to do this?

Reply:
This was done because of the redirect from the `TeamMembersController`. Wasn't sure if removing the `TeamMembersContoller` completely was the way to go. It seemed that the two controllers handled different things. e.g edit permissions vs edit organisation name.